tmk_core/common/chibios/eeprom_stm32.h

```diff
@@ -48,7 +48,7 @@
     #elif defined (MCU_STM32F103RD)
         #define EEPROM_START_ADDRESS    ((uint32_t)(0x8000000 + 384 * 1024 - 2 * EEPROM_PAGE_SIZE))
     #elif defined (MCU_STM32F303CC)
-        #define EEPROM_START_ADDRESS    ((uint32_t)(0x8000000 + 250 * 1024 - 2 * EEPROM_PAGE_SIZE))
+        #define EEPROM_START_ADDRESS    ((uint32_t)(0x8000000 + 256 * 1024 - 2 * EEPROM_PAGE_SIZE))
```

Comment:
Could thes be made as a separate PR?

Reply:
oh odd, I thought this was merged already

Reply:
You probably didn't merge/rebase then.  Should be fine, I think. Unless you wanted to fix it.  Let me know either way.